vagrant/Vagrantfile

```diff
@@ -90,7 +90,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             pm2 logs > /home/vagrant/log/"#{ENV['REPO_NAME']}"/vagrant.log &
           fi
           waitForPM2Daemon
-          pm2 start rackhd-pm2-config.yml
+          cp /home/vagrant/src/build-config/vagrant/rackhd-pm2-config*.yml /home/vagrant/.
+          if (echo "#{ENV['REPOS_UNDER_TEST']}"|grep -q "image-service"); then
```

Comment:
For keep vagrantfile clean, I suggest not to do this ```if``` logic here.
I suggest to put all actions of "if on-image-service" together as possible.

For this yml file, you can use 
```
if image-service
cp rackhd-pm2-config-image-service.yml rackhd-pm2-config.yml
````
out of the vagrantfile.